test/stability/testMerge/testMerge.go

```diff
@@ -195,7 +195,7 @@ func executeTest(configFile, target string, testScenario uint) error {
 		testDoneEvent.Wait()
 		testDoneEvent.L.Unlock()
 
-		composeStatistics()
+		return composeStatistics()
```

Comment:
Good catch! It is really working now?

Reply:
It worked, a bug is that when test actually fails, nil error is returned instead of real error and gotest says all ok, when it actually was not.